### PR TITLE
Build for latest version of SageMath

### PR DESCRIPTION
**Pull request by saraedum**
##### Dependencies

* [ ] We need to figure out how to have Cython build with namespace packages https://trac.sagemath.org/ticket/34187#comment:28

### CODE REVIEW

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender.

File: .ci_support/linux_64_python3.7.____cpythonsagelib9.7.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+libiconv:
+- '1'
+pin_run_as_build:
+  python:
+    min_pin: x.x
+    max_pin: x.x
+python:
+- 3.7.* *_cpython
+sagelib:
+- '9.7'
```
Comment (slel) [inline .ci_support/linux_64_python3.7.____cpythonsagelib9.7.yaml:28]: Does this work despite Sage 9.7 dropping Python 3.7 support?

Comment (saraedum) [reply .ci_support/linux_64_python3.7.____cpythonsagelib9.7.yaml:28]: That build is being skipped by the `skip: True` rule.

Comment (slel) [reply .ci_support/linux_64_python3.7.____cpythonsagelib9.7.yaml:28]: I see, checking what (if anything) goes wrong in that combination
might be precisely the purpose of these two files:

- `.ci_support/linux_64_python3.7.____cpythonsagelib9.7.yaml`
- `.ci_support/osx_64_python3.7.____cpythonsagelib9.7.yaml`

Sorry for the noise.

Comment (saraedum) [reply .ci_support/linux_64_python3.7.____cpythonsagelib9.7.yaml:28]: No worries. These files should not exist but conda-smithy is not (trying to be) smart enough when `skip: True` has a complicated conditional attached as is the case here.

So, an empty CI run happens that immediately terminates successfully.

**Pull request merged**